designPattern/designPattern/Program.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PizzaDesignPattern
+{
+    public interface IPizzaBuilder
+    {
+        public void reset();
+        public void BuildDough();
+        public void BuildSous();
+        public void BuildTopping();
+        public void BuildCheese();
+    }
+    public class WhitePizzaBuilder : IPizzaBuilder
+    {
+        private WhitePizza Pizza = new WhitePizza();
+        public WhitePizzaBuilder()
+        {
+            this.reset();
+        }
+
+        public void BuildDough()
+        {
+            this.Pizza.add("white dough");
+        }
+
+        public void BuildSous()
```

Comment:
It would be nice if you specified some sauces like (_White sauce_, _red sauce_) or what you want rather than just writing the word _sous_.